src/app/config.py

```diff
@@ -47,6 +47,12 @@
 S3_SECRET_KEY: str = os.environ["S3_SECRET_KEY"]
 S3_REGION: str = os.environ["S3_REGION"]
 S3_ENDPOINT_URL: str = os.environ["S3_ENDPOINT_URL"]
+S3_USE_PROXY: bool = os.environ.get("S3_USE_PROXY", "").lower() != "false"
+S3_PROXY_URL: str
+if S3_USE_PROXY:
+    S3_PROXY_URL = os.environ["S3_PROXY_URL"]
+else:
+    S3_PROXY_URL = ""
```

Comment:
Let's use a one line condition:
```suggestion
S3_PROXY_URL: Union[str, None] = os.environ["S3_PROXY_URL"] if S3_USE_PROXY else None
```